### DISCUSSION TITLE
Easiest way to create separate associations plot

### DISCUSSION THREAD

**Discussion by nickjtay** [fbdesignpro/sweetviz#108] [Q&A]
I really like the associations plot because it includes both numerical and categorical features. I looked at the code and it wasn't obvious to me without doing some serious digging, as to how I can split out the plot from the main report.

The problem is that the data supplied to the report has many columns with long column names which overlap one another.

I would love to be able to modify the plot to make it larger, even if it is outside of the SweetViz report. Any suggestions on how I could achieve this with the least amount of work?

**Comment by Zelpheriz** [comment]
Perhaps having the ability to customize and select certain features to only appear on the association graph
